blog/docs/articles/knative-1.0.md

```diff
@@ -0,0 +1,84 @@
+* A command-line client with support for additional feature plugins
+* Moved from an ad-hoc release process to an every 6 week process
+* Support for HTTP2, grpc, and websockets
+* Introduced Brokers and Triggers to simplify publishing and subscribing to events while decoupling producers and consumers
+* Support for Eventing components delivering to non-Knative components, including off-cluster components or specific URLs on a host
+* Support for automatic provisioning of TLS certificates (either via DNS or HTTP01 challenges)
+* Custom delivery options for event destinations, including retries and dead-letter queues for undeliverable messages
+* Event tracing support for both Brokers and Channels, for improved debugging
+* [The Tekton project](https://tekton.dev/), spawned from Knative Build
+* Parallel and Sequence components, to codify certain composite eventing workflows
+* Documentation for Event Sources and how to contribute them, currently covering about 40 different event sources
+* "Hitless” upgrades with no dropped requests between minor version releases
+* Redesign of API shapes for Serving to match the PodTemplateSpec used by Deployment, CronJob, etc, to simplify usage for Kubernetes-aware users
+* Support for injecting event destination addresses into PodTemplateSpec shaped objects
+* Support for horizontal pod autoscaling based on concurrency or RPS
+* High availability of control plane components using leader election slices
+* An operator, to help administrators install Knative
+* A quickstart, for developers to try Knative locally
+* Simplified management and publication of Services using DomainMapping
+
+## Learn more
+Knative steering committee member Ville Aikas is [the guest on the Kubernetes Podcast from Google this week](https://kubernetespodcast.com/episode/166-knative-1.0/), telling the story of the creation of the project and its journey to 1.0.  You can also join [the Knative community meetup on November 17,](https://calendar.google.com/calendar/u/0/r/eventedit/NnAycjJyZmdlMTF1b2FuOGJzZjZ1dXA0aTZfMjAyMTExMjRUMTczMDAwWiBrbmF0aXZlLnRlYW1fOXE4M2JnMDdxczViOXJyc2xwNWpvcjRsNnNAZw?tab=mc) where Ville will talk about the latest changes to the project.
+
+## Get involved
+Newcomers to the Knative community are always welcome! [Join the Knative Slack space](https://slack.knative.dev/) to ask questions and troubleshoot issues as you get acquainted with the project. Finally, find all the [project documentation](https://knative.dev/docs/) on the Knative website and [contribute to the project on GitHub](https://github.com/knative).
+
+## Thank you to our contributors!
+Achieving this milestone has truly been a community effort — we’d be remiss not to thank some of the people who helped us get here.
+Thanks to…
+
+* Knative Supporting Companies
+  * Google
+  * IBM
+  * Red Hat
+  * SAP
+  * TriggerMesh
+  * VMware
```

Comment:
These aren't showing as indented properly (see https://deploy-preview-4402--knative.netlify.app/blog/articles/knative-1.0/)

Reply:
You may also like to link to their respective Knative-powered services?